xml/System.IO.Enumeration/FileSystemName.xml

```diff
@@ -78,12 +78,17 @@
         <Parameter Name="ignoreCase" Type="System.Boolean" />
       </Parameters>
       <Docs>
-        <param name="expression">To be added.</param>
-        <param name="name">To be added.</param>
-        <param name="ignoreCase">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="expression">The expression to match with, such as "*.foo".</param>
+        <param name="name">The name to check against the expression.</param>
+        <param name="ignoreCase"><see langword="true" /> to ignore case (default), <see langword="false" /> if the match should be case-sensitive.</param>
+        <summary> Verifies if the given Win32 expression matches the given name. Supports the following wildcards: '*', '?', '&lt;', '&gt;', '"'. The backslash character '\' escapes.</summary>
+        <returns><see langword="true" /> if the given expression matches the given name, <see langword="false" /> otherwise.</returns>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+This is based off of System.IO.PatternMatcher used in FileSystemWatcher, which is based off of RtlIsNameInExpression, which defines the rules for matching DOS wildcards ('*', '?', '&lt;', '&gt;', '"').
```

Comment:
`System.IO.PatternMatcher` doesn't exist anymore and was internal to begin with, so we shouldn't mention it.

Reply:
Ok I removed the remark.